templates/fargate-service.cfn.yml

```diff
@@ -0,0 +1,682 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+# A CloudFormation template to deploy an additional service to Fargate. This requires an existing
+# cluster deployed by fargate.cfn.yml.
+
+Description: Fargate Service
```

Comment:
IIRC, a Fargate service is also part of the Fargate template. Do you intend to DRY this up and use this template within a the "other" Fargate template? I notice several things in `fargate.cfn.yml` changed to mirror decisions you made in this file.

Reply:
Ha, I have been thinking about ways to DRY this up. Listener rules require a default target group, so I need to consider this. I can add a couple of conditions to a single template, but I am not sure if the result is going to be difficult for most to read. I created a ticket to investigate.

Reply:
Cool. Seems good enough for now.